patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -77,15 +132,29 @@
          float f = this.func_110143_aJ();
  
          if (f > 0.0F)
-@@ -839,6 +847,7 @@
+@@ -831,14 +851,21 @@
+     {
+         return ((Float)this.field_70180_af.func_187225_a(field_184632_c)).floatValue();
+     }
++    
+ 
+     public void func_70606_j(float p_70606_1_)
+     {
++    	setLastHealth(func_110143_aJ());
+         this.field_70180_af.func_187227_b(field_184632_c, Float.valueOf(MathHelper.func_76131_a(p_70606_1_, 0.0F, this.func_110138_aP())));
+     }
++    
++    public float getLastHealth() { return ((Float)this.field_70180_af.func_187225_a(LAST_HEALTH)).floatValue(); }
```

Comment:
Do you even look at this before you commit?

Reply:
I look at the java code, am I supposed to look at the patch?